frame/im-online/src/lib.rs

```diff
@@ -219,19 +222,13 @@ pub struct Heartbeat<BlockNumber>
 	pub authority_index: AuthIndex,
 }
 
-pub trait Trait: frame_system::Trait + pallet_session::historical::Trait {
+pub trait Trait: SendTransactionTypes<Call<Self>> + pallet_session::historical::Trait {
```

Comment:
How does this work? where did system go?

Reply:
`pallet_session::historical_trait`'s super::super trait is `frame_system::trait`

Reply:
Previously, `SendTransactionTypes`'s super trait was `SigningTypes` which in turn had the super trait of `frame_system::trait` but i removed this requirement to make submitting raw transactions easier by not requiring im-online to define `AppCrypto`.